tests/testthat/test-license_note.R

```diff
@@ -0,0 +1,13 @@
+test_that("LICENSE.note", {
```

Comment:
Probably a more informative name for the test `"LICENSE.note is generated properly"`